<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sign</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>录入信息</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelName">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>姓名：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditName">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelGender">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>性别：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxGender">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="editable">
             <bool>true</bool>
            </property>
            <item>
             <property name="text">
              <string>男</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>女</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelPhone">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>手机号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditPhone">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelBirth">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>生日：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditBirth">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>日期：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateTimeEdit">
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonOK">
            <property name="font">
             <font>
              <family>STFangsong</family>
             </font>
            </property>
            <property name="text">
             <string>确定</string>
            </property>
            <property name="shortcut">
             <string>Meta+Return</string>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableViewChoose">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="dragEnabled">
           <bool>false</bool>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>统计信息</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUpdate"/>
   <addaction name="actionQuery"/>
   <addaction name="actionQueryPeople"/>
   <addaction name="actionQueryBirthday"/>
   <addaction name="actionQueryPhone"/>
   <addaction name="actionQueryAll"/>
   <addaction name="actionClearEdits"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>更新字典表</string>
   </property>
  </action>
  <action name="actionQuery">
   <property name="text">
    <string>按日期查询</string>
   </property>
  </action>
  <action name="actionQueryPeople">
   <property name="text">
    <string>按姓名查询</string>
   </property>
  </action>
  <action name="actionQueryPhone">
   <property name="text">
    <string>按手机号查询</string>
   </property>
  </action>
  <action name="actionClearEdits">
   <property name="text">
    <string>清理输入框</string>
   </property>
  </action>
  <action name="actionQueryBirthday">
   <property name="text">
    <string>按生日查询</string>
   </property>
  </action>
  <action name="actionQueryAll">
   <property name="text">
    <string>遍历所有</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditName</tabstop>
  <tabstop>comboBoxGender</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>lineEditBirth</tabstop>
  <tabstop>dateTimeEdit</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>tableViewChoose</tabstop>
  <tabstop>tableView</tabstop>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
